client/capstone-app/src/component/DocInfo.js

```diff
@@ -1,107 +1,105 @@
-function Docdata() {
-  let data = [
-    {
-      Dcoimg:
-        "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS6oT9DZBnkbcGnMMQyARlpIPHFbuxeGihowVuI-8V_RA&usqp=CAU&ec=48600112",
-      name: "qwertyu",
-      specialist: "Cardiologist",
-      timings: "9:00 am to 12:30 pm",
-      count:0
-    },
-    {
-      Dcoimg:
-        "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS6oT9DZBnkbcGnMMQyARlpIPHFbuxeGihowVuI-8V_RA&usqp=CAU&ec=48600112",
-      name: "asdfghj",
-      specialist: "Cardiologist",
-      timings: "9:00 am to 12:30 pm",
-      count:0
-    },
-    {
-      Dcoimg:
-        "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS6oT9DZBnkbcGnMMQyARlpIPHFbuxeGihowVuI-8V_RA&usqp=CAU&ec=48600112",
-      name: "asdfghjk",
-      specialist: "Cardiologist",
-      timings: "9:00 am to 12:30 pm",
-      count:0
-    },
-    {
-      Dcoimg:
-        "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS6oT9DZBnkbcGnMMQyARlpIPHFbuxeGihowVuI-8V_RA&usqp=CAU&ec=48600112",
-      name: "qazxsde",
-      specialist: "Cardiologist",
-      timings: "9:00 am to 12:30 pm",
-      count:0
-    },
-    {
-      Dcoimg:
-        "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS6oT9DZBnkbcGnMMQyARlpIPHFbuxeGihowVuI-8V_RA&usqp=CAU&ec=48600112",
-      name: "plkmnjhy",
-      specialist: "Cardiologist",
-      timings: "9:00 am to 12:30 pm",
-      count:0
-    },
-    {
-      Dcoimg:
-        "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS6oT9DZBnkbcGnMMQyARlpIPHFbuxeGihowVuI-8V_RA&usqp=CAU&ec=48600112",
-      name: "vcdfrgthnbvf",
-      specialist: "Cardiologist",
-      timings: "9:00 am to 12:30 pm",
-      count:0
-    },
-  ];
-  return data;
-}
-export default function Doclist() {
+// import { Link } from "react-router-dom";s
+
+export default function DocInfo() {
+  const {id} = useParams()
+  // console.log(id)
   const [timing, setTiming] = useState("");
   const [name, setName] = useState("");
-  const [docData,setDocData]=useState(Docdata())
-  const [num, setNum] = useState(0);
   const[data,setdata] = useState([]);
+  // const[counts,setCounts]=useState('');
+  const[counts,setCounts]=useState(0);
+  const [docData,setDocData]=useState(data)
+  // const [num, setNum] = useState(0);
+  // value={num} onChange={(event)=>setNum(event.target.value)}
   
-  useEffect(()=>{
-    localStorage.setItem("Num", num);
-  },)
+  // useEffect(()=>{
+  //   localStorage.setItem("Num", num);
+  // },)
     
   useEffect(()=>{
     fetch("http://localhost:2917/User")
         .then((response) => response.json())
         .then((data) => {
           // console.log(data);
           setdata(data);
+          // setCounts(data.Count);
         })
         .catch((error) => {
           console.log(" failed to fetch");
         })
-  },[])
-  
- 
+  },[id])
+  // /update/:id
+const UpdatCount= (e,count)=>{
+  console.log(e.target.id)
+   fetch(`http://localhost:2917/update/${e.target.id}`,{
```

Comment:
Add `http://localhost:2917` in `.env` and use here using `process.env`, later when you have to deploy your application it becomes simpler for you.